proto/public/alertmanager/v3/status.proto

```
@@ -0,0 +1,93 @@
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package alertmanager.v3;
```

Buf (1.70.0)

[error] Files with package "alertmanager.v3" must be within a directory "alertmanager/v3" relative to root but were in directory ".".

(PACKAGE_DIRECTORY_MATCH)